packages/eas-cli/src/submissions/utils/errors.ts

```diff
@@ -12,13 +14,19 @@ enum SubmissionErrorCode {
   IOS_UNKNOWN_ERROR = 'SUBMISSION_SERVICE_IOS_UNKNOWN_ERROR',
   IOS_MISSING_APP_ICON = 'SUBMISSION_SERVICE_IOS_MISSING_APP_ICON',
   IOS_INVALID_SIGNATURE = 'SUBMISSION_SERVICE_IOS_INVALID_SIGNATURE',
+  IOS_INCORRECT_CREDENTIALS = 'SUBMISSION_SERVICE_IOS_INVALID_CREDENTIALS',
 }
 
 const SubmissionErrorMessages: Record<SubmissionErrorCode, string> = {
   [SubmissionErrorCode.ARCHIVE_DOWNLOAD_NOT_FOUND_ERROR]:
     "Failed to download the archive file (Response code: 404 Not Found). Please make sure the URL you've provided is correct.",
   [SubmissionErrorCode.ARCHIVE_DOWNLOAD_FORBIDDEN_ERROR]:
     'Failed to download the archive file (Response code: 403 Forbidden). This is most probably caused by trying to upload an expired build artifact. All Expo build artifacts expire after 30 days.',
+  [SubmissionErrorCode.ARCHIVE_EXTRACT_CORRUPT_ARCHIVE_ERROR]:
+    'The compressed archive is corrupted or has an invalid format. EAS Submit accepts artifacts files provided directly or compressed into tar.gz archives.',
```

Comment:
```suggestion
    'The compressed archive is corrupted or has an invalid format. Valid formats include .apk/.aab/.ipa files, or one of these files compressed into a tar.gz archive.',
```

Reply:
this wording was kind of difficult, maybe @EvanBacon or @ide have ideas for how to phrase it better